app/src/main/AndroidManifest.xml

```diff
@@ -17,7 +19,7 @@
         android:supportsRtl="true">
         <meta-data
             android:name="com.google.android.geo.API_KEY"
-            android:value="AIzaSyCJS0UNcCJom-elHhFSMQQI5ARRmbQTQAA" />
```

Comment:
Please do not change the API key as because it will vary from machine to machine